dags/datasets/earlyon-centres-registry.py

```diff
@@ -0,0 +1,442 @@
+        "ward": "In which ward  (number) is this location",
+        "ward_name": "The ward name of the location",
+        "located_in_school": "Is this Location in a school (Yes/No)?",
+        "school_name": "Name of School EarlyON is located in",
+        "lat": "Latitudinal coordinates of the location address",
+        "lng": "Longitudinal coordinates of the location address",
+        "website": "the location website where parents and caregivers can find more information about the programs",
+        "website_name": "Agency's Website",
+        "website_description": "Agency's Website",
+        "consultant_fullname": "The first and last name of the EarlyOn Consultant assigned to this location",
+        "consultant_phone": "The contact phone number of the EarlyOn Consultant assigned to this location",
+        "email": "The contact email for the EarlyOn Consultant assigned to this location",
+        "contact_fullname": "The contact email for the EarlyOn Consultant assigned to this location",
+        "contact_title": "The contact email of the Location Administrator",
+        "phone": "The contact phone number of the Location Administrator ",
+        "contact_email": "The contact email of the Location Administrator",
+        "dropinHours": "Day of the week, Start and end time, whether program is full time, and frequency for in-person programs",
+        "registeredHours": "Day of the week, Start and end time, whether program is full time, and frequency for in-person registered programs",
+        "virtualHours": "Day of the week, Start and end time, whether program is full time, and frequency for virtual programs",
+        "geometry":"markers geometry",
+        "centre_type": "Centre",
+        "program": "<a id='13650' href='#modal13650_' data-toggle='modal' data-target='#myModal' onclick='fillData(13650)'>101 Spruce St EarlyON Child and Family Centre</a>",
+}
+delete_col = ['program', 'centre_type']
+EXPECTED_COLUMNS = [x for x in (m for m in maker_attr_def) if x not in delete_col]
+
+def send_failure_message():
+    airflow_utils.message_slack(
+        name=PACKAGE_NAME,
+        message_type="error",
+        msg="Job not finished",
+        active_env=Variable.get("active_env"),
+        prod_webhook=Variable.get("active_env") == "prod",
+    )
+
+
+with DAG(
+    dag_id,
+    default_args = airflow_utils.get_default_args(
+        {
+            "owner": "Gary Qi",
+            "depends_on_past": False,
+            "email": ["gary.qi@toronto.ca"],
+            "email_on_failure": False,
+            "email_on_retry": False,
+            "retries": 0,
+            "on_failure_callback": task_failure_slack_alert,
+            "start_date": days_ago(1),
+        }
+    ),
+    description="Take new earlyon.json from opendata.toronto.ca and put into datastore",
+    schedule_interval="0 18 * * *",
+    catchup=False,
+    tags=["earlyon","gary"],
```

Comment:
add "datasets" to the tags - so we know quickly (on the airflow interface) this isnt a `sustainment` dag

Reply:
Updated tag and changed schedule